<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MeshGui::RemoveComponents</class>
 <widget class="QDialog" name="MeshGui::RemoveComponents" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Remove components</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Deselect</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spDeselectComp" />
      </item>
      <item row="0" column="0" >
       <widget class="QPushButton" name="deselectRegion" >
        <property name="text" >
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="deselectAll" >
        <property name="text" >
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="deselectComponents" >
        <property name="text" >
         <string>Components</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>211</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Select</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spSelectComp" />
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="selectComponents" >
        <property name="text" >
         <string>Components</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QPushButton" name="selectRegion" >
        <property name="text" >
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="selectAll" >
        <property name="text" >
         <string>All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>selectRegion</tabstop>
  <tabstop>selectAll</tabstop>
  <tabstop>selectComponents</tabstop>
  <tabstop>spSelectComp</tabstop>
  <tabstop>deselectRegion</tabstop>
  <tabstop>deselectAll</tabstop>
  <tabstop>deselectComponents</tabstop>
  <tabstop>spDeselectComp</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MeshGui::RemoveComponents</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
